Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/Persona.java

```diff
@@ -469,6 +481,18 @@ public PersonaAlias addAlias(String alias, String justification, Persona.Confide
     public void removeAlias(PersonaAlias alias) throws CentralRepoException {
         PersonaAlias.removePersonaAlias(alias);
     }
+    
+    /**
```

Comment:
Per our coding guidelines (shared with you), the method documentation should follow standard English capitalization and periods etc. 

I wont hold up this PR, but if you can address these, and keep in mind future, that would be great.